deployer.go

```diff
@@ -105,12 +104,11 @@ func main() {
 		panic(err)
 	}
 
-	for {
-		log.Printf("INFO Starting deploy run")
-		deployAndRecover(d)
-		time.Sleep(time.Duration(*intervalInSecondsBetweenDeploys) * time.Second)
-		log.Printf("INFO Finished deploy run")
+	log.Printf("INFO Starting deploy run")
+	if err := d.deployAll(); err != nil {
+		panic(err)
 	}
+	log.Printf("INFO Finished deploy run")
 }
 
 func deployAndRecover(d *deployer) {
```

Comment:
Yeah, same logic applies, run runs in a loop, only the loop is external to the app